model/provider_location.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class ProviderLocation < Sequel::Model
+  include ResourceMethods
```

Comment:
If we only plan on adding locations inside migrations, using `plugin :static_cache` would be a good idea.

Reply:
The idea is to dynamically be able to add locations at runtime, that's why we cannot use `static_cache`